src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -2422,34 +2422,31 @@ private void LexSyntaxTrivia(bool afterFirstToken, bool isTrailing, ref SyntaxLi
 
         private bool IsConflictMarkerTrivia()
         {
-            var position = TextWindow.Position;
-            var text = TextWindow.Text;
-            if (position == 0 || SyntaxFacts.IsNewLine(text[position - 1]))
```

Comment:
can you keep the logic the same, but just use the sliding window?  this seems more complex to understand.

Reply:
The logic is the same. Which specific part do you feel is more complex?

Reply:
the logic before was to check that we were after a newline.  Now the logic checks the character we are on first.  I'd prefer to keep the old logic as it makes the most sense to me.  These markers must be after newlines, so that makes sense as the first thing to check.